src/modules/Admin/Components/Home/Components/Home.tsx

```diff
@@ -1,7 +1,178 @@
-import React from 'react'
+import * as React from 'react';
+import axios from "axios";
+import WorkOutlineIcon from '@mui/icons-material/WorkOutline';
+import { Doughnut } from 'react-chartjs-2';
+import { Chart as ChartJS, ArcElement, Tooltip, Legend } from 'chart.js';
+import { toast } from 'react-toastify';
+
+ChartJS.register(ArcElement, Tooltip, Legend);
 
 export default function Home() {
+  const [dashboardData, setDashboardData] = React.useState<any>(null);
+
+  const Submit = async () => {
+    try {
+      const response = await axios.get(`https://upskilling-egypt.com:3000/api/v0/admin/dashboard`, {
+        headers: {
+          Authorization: `Bearer ${localStorage.getItem("token")}`,
```

Comment:
We can create an axios interceptor, and handle the token inside it. This will eliminate the need to write this line in each request we make.